sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/query.py

```diff
@@ -0,0 +1,278 @@
+                    bigquery.SchemaField("app", "STRING"),
+                    bigquery.SchemaField("app_token", "STRING"),
+                    bigquery.SchemaField("network", "STRING"),
+                    bigquery.SchemaField("network_token", "STRING"),
+                    bigquery.SchemaField("campaign", "STRING"),
+                    bigquery.SchemaField("campaign_token", "STRING"),
+                    bigquery.SchemaField("adgroup", "STRING"),
+                    bigquery.SchemaField("adgroup_token", "STRING"),
+                    bigquery.SchemaField("creative", "STRING"),
+                    bigquery.SchemaField("creative_token", "STRING"),
+                    bigquery.SchemaField("country", "STRING"),
+                    bigquery.SchemaField("os", "STRING"),
+                    bigquery.SchemaField("device", "STRING"),
+                    bigquery.SchemaField("clicks", "INTEGER"),
+                    bigquery.SchemaField("installs", "INTEGER"),
+                    bigquery.SchemaField("limit_ad_tracking_install_rate", "FLOAT"),
+                    bigquery.SchemaField("click_conversion_rate", "FLOAT"),
+                    bigquery.SchemaField("impression_conversion_rate", "FLOAT"),
+                    bigquery.SchemaField("sessions", "INTEGER"),
+                    bigquery.SchemaField("daus", "INTEGER"),
+                    bigquery.SchemaField("waus", "INTEGER"),
+                    bigquery.SchemaField("maus", "INTEGER"),
+                ],
+            )
+
+            # Table names are based on the app name seen in the Adjust dashboard"
+            destination = f"{project}.{dataset}.adjust_deliverables_v1_${partition}"
+
+            job = client.load_table_from_file(f_csv, destination, job_config=job_config)
+
+            print(
+                f"Writing adjust data for all apps to {destination}. BigQuery job ID: {job.job_id}"
+            )
+            job.result()
+
+
+def main():
+    """Input data, call functions, get stuff done."""
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument("--date", required=True)
+    parser.add_argument("--project", default="moz-fx-data-shared-prod")
+    parser.add_argument("--dataset", default="adjust_derived")
+
+    args = parser.parse_args()
+
+    app_list = json.loads(APP_TOKEN_LIST)
+
+    # Cycle through the apps to get the relevant kpi data
+    for app in app_list:
+        print(f'This is data for {app["app_name"]} - {app["app_token"]}')
+        # Ping the Adjust URL and get a response
+        json_file = download_adjust_kpi_data(args.date, API_TOKEN, app["app_token"])
+
+        data = []
+
+        query_export = check_json(json_file.text)
+
+        if query_export is not None:
+            # This section writes the tmp json data into a temp CSV file which will then be put into a BigQuery table
+            data = clean_json(query_export)
+            print("finished json_read")
```

Comment:
In the current form the log may not be helpful, if you think it's useful to have it try to make the message more self contained. json_read finished for? Is this good? Generally, instead of print I'd suggest using the `logging` library as it provides different log levels such as `debug`, `info`, `warning`, and `error` which helps to indicate if this log is just informational or does it suggest something is off.